frontend/src/js/prime-web-receiver-pre.js

```diff
@@ -147,7 +147,7 @@ function idleTimer() {
  *  logged in, directs to /daily-data page
  */
 function login() {
-    const OKTA_clientId = '0oa6fm8j4G1xfrthd4h6';
+    const OKTA_clientId = process.env.OKTA_clientId || '0oa6kt4j3tOFz5SH84h6';
```

Comment:
If you wanted, you could default the ID OKTA_clientId `''` and then runtime check if it's not set correctly. If it's missing, you could throw an error or `console.error()` that it's not set in the env file.

My thought experiment:  The oka id changes for the `test` env at some point in the future (using a test okta account would allow for safer experimentation)... but if it's miss configured in the test .env, then the code defaults to the _production_ id in code (which is less safe). 

So, it seems better it fails with a clear message.

Reply:
This is for the 11ty site, which is only in production as I understand it, hence the hard-coded production string fallback. Shockingly, there isn't a .env file present in that directory, but there are a few `process.env` calls being made. I'm guessing it's something we didn't check into the repo. Regardless though, this is merely patchwork for the soon-to-be-deprecated site.

Reply:
Ohhh. Thanks for the clarification!